### ISSUE TITLE
AutoPlay not fired, in first load

### ISSUE THREAD

**Issue by safeimuslim** [akiran/react-slick#908]
Hi, can you help me? I have problem with auto play.

AutoPlay not fired, in first load. I must manual fired and then slider work properly.

SliderContent component call in render.

```
/** define slider content */
const SliderContent = ({ sliders, banners, height, width }) => {
  const hasBanner = banners.data.length > 0
  const listImages = banners.data.map((banner, index) => (
    <div key={index} style={{ width, height: height / 3, backgroundImage: `url(${banner.image})`, backgroundPosition: 'center', backgroundSize: 'cover' }} />
  ))
  if (!hasBanner) return null
  return (
    <Section>
      <Content className='slide-banner'>
        {
          <Slider {...sliders}>
            { listImages }
          </Slider>
        }
      </Content>
    </Section>
  )
}
```

**Comment by Schmerb** [comment]
I have this same issue.
`const settings = {
			dots: true,
			infinite: false,
			speed: 1500,
			autoplaySpeed: 2500,
			autoplay: true,
			slidesToShow: 1,
			slidesToScroll: 1,
			nextArrow: <NextArrow />,
			prevArrow: <PrevArrow />,
			beforeChange: this.beforeChange,
			afterChange: this.afterChange,
			responsive: [
				{
					breakpoint: 650,
					settings: {
						arrows: false
					}
				}
			]
		};
return (
			<Slider {...settings} >
				{currencySlides}
			</Slider>
		);
`

**Comment by SCasarotto** [comment]
I am also experiencing this issue.

**Comment by ruppalsingh** [comment]
@safeimuslim After you manually change slides once, does the autoplay work properly or does it stop working at any point after that?

**Comment by Schmerb** [comment]
My slides appear to work initially, but once I hover over a slide or
interact (click, drag, go to another slide) there is a delay. Sometimes the
delay is only 2-3x as long as the autoplaySpeed I set it to and other times
it stops completely.

If my cursor is hovering / moving over a current slide, the slides do not
move again until I stop moving the cursor around or move the cursor off of
the slide.

**Comment by safeimuslim** [comment]
@ruppalsingh work properly

**Comment by safeimuslim** [comment]
Fixed => upgrade react-slick version 0.14.0 to 0.16.0

**Comment by ukreddy** [comment]
I have the same issue while developing SPFx with Carousel and i have verified that my react-slick version is 0.23.2, any suggestions?
